README.md

```diff
@@ -61,7 +61,7 @@ Likely causes:
 Likely causes:
 - Secret key is not set.
 - Request was made with at least one empty parameter.
-- `signature` is not the last parameter in the parameters list.
+- `timestamp`&`signature` is not the last parameter in the parameters list.
```

Comment:
The `Likely causes:` section refers to error message `Signature for this request is not valid.`, which is concentrated on Signature, therefore shouldn't be related with the timestamp order.

Do you have a reproducible example we can check on to see the behavior? So that we can be sure of your use case.

Reply:
The phrase is not grammatically correct. :)